VERSION

```diff
@@ -1 +1 @@
-0.32.4
+0.32.5
```

Comment:
This should not be done manually... I think! version ci pipepline will bump the version depending on the changelog's contents